### ISSUE TITLE
Setting ifa enabled:false breaks application

### ISSUE THREAD

**Issue by danwenzel** [adopted-ember-addons/ember-cli-ifa#148]
Setting ifa `enabled` to `false` causes the app to not be able to boot.

```js
// environment.js
ifa: {
  enabled: false,
},
```
```js
// ember-cli-build.js
fingerprint: {
  enabled: true, // or false; doesn't matter
  generateAssetMap: true
}
```
Get the following error in the browser when booting the application:
```console
Uncaught (in promise) SyntaxError: Unexpected token _ in JSON at position 0
```

![image](https://user-images.githubusercontent.com/11724146/77318324-e3e00080-6cda-11ea-9bb8-837be97655b8.png)

If I'm reading the code correctly, this is happening because:

- The meta placeholder string is appended here: https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/index.js#L38

- but is never replaced because ifa is not set to enabled: https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/index.js#L61-L63

- and then the initializer tries to `JSON.parse` the bad string here: https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/addon/utils/get-asset-map-data.js#L17

**Comment by xuanluong** [comment]
Hi @danwenzel,

I came across your issue. I'm not sure if you have it resolved yet?

IIUC, if this content of the meta tag was actually `<meta name="ember-cli-ifa:assetMap">` then the value of `metaTag.content` in https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/addon/utils/get-asset-map-data.js would be an empty string and `JSON.parse` would not have been called.

The error message of `Unexpected _ token` seems to suggest that the meta tag was containing the placeholder value (`'__ember-cli-ifa__AssetMapPlaceholder__'`), which is defined here: https://github.com/adopted-ember-addons/ember-cli-ifa/blob/master/index.js#L6

I'm curious whether you are having this issue in production or in development and whether the setting 
```
// environment.js
ifa: {
  enabled: false,
},
```
is applied in production or in development?

**Comment by danwenzel** [comment]
Thanks for the nudge, @xuanluong ! It looks like this issue was resolved via `v0.10.0`.  Closing
